src/proj_symbol_rename.h

```diff
@@ -72,11 +72,14 @@
 #define proj_context_set_sqlite3_vfs_name                                      \
     internal_proj_context_set_sqlite3_vfs_name
 #define proj_context_set_url_endpoint internal_proj_context_set_url_endpoint
+#define proj_context_set_user_writable_directory                               \
+    internal_proj_context_set_user_writable_directory
 #define proj_context_use_proj4_init_rules                                      \
     internal_proj_context_use_proj4_init_rules
 #define proj_convert_conversion_to_other_method                                \
     internal_proj_convert_conversion_to_other_method
 #define proj_coord internal_proj_coord
+#define proj_coordinate_metadata_create internal_proj_coordinate_metadata_create
```

Comment:
This was not created before because the script was not executed for some time, right? Was it risky?

Reply:
> This was not created before because the script was not executed for some time, right? 

yes

Was it risky?

moderately. This is only used on special builds with -DPROJ_RENAME_SYMBOLS as used in the GDAL Docker image to have a custom PROJ build used only by GDAL, while allowing other packaged libs to use the system libproj.